src/main/scala/co/theasi/plotly/Plot.scala

```diff
@@ -263,6 +263,15 @@ object CartesianPlot {
   *   .withSurface(zs2, SurfaceOptions().name("bottom"))
   * }}}
   *
+  * ==3D Scatter plots==
+  *
+  * Create 3D scatter `ThreeDPlot` instances using `withScatter3D`:
+  * {{{
+  * val xs = Vector(1.0, 2.0)
+  * val ys = Vector(4.0, 5.0)
+  * val zs = Vector(-1.0, -3.0)
+  * val plot = ThreeDPlot().withScatter3D(xs, ys, zs, ScatterOptions())
```

Comment:
If we are using the default, we probably don't need to specify `ScatterOptions` explicitly in the example, to keep it as simple as possible? 

People who want to know more about the options can look at the API docs for `withScatter3D`?